.eslintrc

```diff
@@ -1,6 +1,19 @@
 {
-  "extends": "standard",
+  "plugins": [
+    "node"
+  ],
+  "extends": [
+    "eslint:recommended",
+    "plugin:node/recommended"
+  ],
+  "parserOptions": {
+    "ecmaVersion": 2017
+  },
+  "env": {
+    "node": true,
+    "es6": true
+  },
   "rules": {
-    "no-new-func": "off"
+    "no-unused-vars": 0
```

Comment:
Are there a lot of these for good reasons?

Reply:
Not really. If you enable it there's a handful of event handlers that don't use the `msg` parameter:

```text
lib/client.js
  283:39  error  'msg' is defined but never used  no-unused-vars
  288:34  error  'msg' is defined but never used  no-unused-vars
  300:37  error  'msg' is defined but never used  no-unused-vars
  306:38  error  'msg' is defined but never used  no-unused-vars

lib/query.js
  220:49  error  'connection' is defined but never used  no-unused-vars
```

I think I held off to keep the incremental diffs minimal and not touch the event handlers. Lemme take a peek as I think it's just a couple ignore tag comments to enable it and would be nice going forward.